tests/e2e/react-router-user-journeys.spec.ts

```diff
@@ -232,7 +234,7 @@ test.describe('React Router user journeys', () => {
 
       await fetch(`${edgeSite.url}/purge-cdn?tag=cached-for-a-year-tag`)
 
-      await page.waitForTimeout(1000)
+      await page.waitForTimeout(PURGE_BUFFER_MS)
```

Comment:
Now knowing the true source of the issues, we might be able to revert some of these timing changes